### DISCUSSION TITLE
[feature request] Automatically create multi-output device on install

### DISCUSSION THREAD

**Discussion by alectrocute** [ExistentialAudio/BlackHole#315] [Feature Requests]
Hi BH devs! Love this project.

I think it would solve a huge user pain point if we created the multi-output device upon installation of BH.

Perhaps this multi-output device, aptly named “BlackHole Loopback” would have both channel count devices and the default system output device enabled inside of it by default.

https://web.archive.org/web/20140716012404/http://daveaddey.com/?p=51

Thanks!

**Comment by devinroth** [comment]
Not a bad idea. I think making an option for users during installation would work great. Creating a script to do it isn’t hard.

Devin Roth

**Comment by MaikWaschfeld** [comment]
And switch on "Drift Correction" by default, if that's scriptable,.because no consumer/normal user knows what that is for.

Maik Waschfeld

**Comment by devinroth** [comment]
It is scriptable. I’ve done it before. And yup, multi-outputs / aggregates are the #1 reason people have issues.

Devin Roth
